### PR TITLE
feat(color): add `--no-color` flag

### LINKED ISSUE #5294: Add flag to disable color output
### New feature motivation

When running prowler in some automated environments (e.g. Jenkins) it's too hard to read the tool output because of ANSI escape symbols. If there were a flag, e.g. `--no-color` to disable this behavior it would be great!
image

### Solution Proposed

Add a flag to disable the color output, e.g. `--no-color` 

### Describe alternatives you've considered

There is a Jenkins AnsiColor plugin https://plugins.jenkins.io/ansicolor/

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by MrSecure**
### Context

Fix #5294 

### Description

- Import colorama's `init`
- Add `--no-color` command line option (default `False`)
- Use `no_color` option to control colorama init.

### Checklist

- Are there new checks included in this PR? **No**
- [x] Review if the code is being covered by tests.
  - Tests added to the parser testing library
- [x] Review if code is being documented following this specification https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings
- [x] Review if backport is needed.

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by MrCloudSec** [comment]
Hi @MrSecure, thanks a lot for your contribution! We will review this asap.

**Comment by MrSecure** [comment]
Should the `--only-logs` flag set `--no-color` in addition to `--no-banner` ?
Or does that complicate things since `--only-logs` is slated for removal in v5?

**Comment by jfagoagas** [comment]
> Should the `--only-logs` flag set `--no-color` in addition to `--no-banner` ? Or does that complicate things since `--only-logs` is slated for removal in v5?

The `--only-logs` flag removes all the pure CLI output so there is no need to set also `--no-color` and it is going to be deprecated in v5 since it is no longer needed because Prowler is usable as an SDK.

Regarding the `--no-banner`, I think they should work independently.

File: prowler/__main__.py [hunk 1]
```diff
@@ -112,6 +113,11 @@ def prowler():
         and not checks_folder
     )
 
+    if args.no_color:
+        colorama_init(strip=True)
+    else:
+        colorama_init(strip=False)
```
Comment (pedrooot) [inline prowler/__main__.py]:
```suggestion
```
Is this needed?

**Review by pedrooot — CHANGES_REQUESTED** [review]
Thanks for your contribution! Please review my comments 🙏🏼

File: prowler/__main__.py (hunk 1 above)
Comment (MrSecure) [reply prowler/__main__.py]: There's a good chance it's not strictly required. However, as the default is 'None' with an OS-dependent behavior I thought it was wise to keep.  Reviewing the OS-dependent behavior, I agree, this should be remove.

Comment (MrSecure) [reply prowler/__main__.py]: fixed.

Comment (pedrooot) [reply prowler/__main__.py]: Thanks! I'll add to your PR info about this flag inside the docs, thanks for your contribution!

**Comment by jfagoagas** [comment]
Hello @MrSecure @pedrooot I've just removed the `-Q` option since for now we only want to add long flags to the CLI. 

Thank you!

**Review by pedrooot — APPROVED** [review]
🔝

**Comment by toniblyx** [comment]
Thank you @MrSecure :D

**Pull request merged**